website/settings/defaults.py

```diff
@@ -1981,26 +1981,16 @@ def from_node_usage(cls,  usage_bytes, private_limit=None, public_limit=None):
         """ This should indicate if a node is at or over a certain storage threshold indicating a status. If nodes have
```

Comment:
Nit: Can probably get rid of the bit about "If nodes have a custom limit this should indicate that". Custom limit is already indicated on the node itself, anyway.